tests/click_tests/test_autolock.py

```diff
@@ -110,10 +110,10 @@ def test_autolock_passphrase_keyboard(device_handler: "BackgroundDeviceHandler")
     assert layout.text == "< PassphraseKeyboard >"
 
     CENTER_BUTTON = buttons.grid35(1, 2)
-    # total sleep time: 11 * 1.1 = 12.1 seconds
+    # total sleep time: 11 * 1.5 = 16.5 seconds
```

Comment:
the intention here was to keep clicking for long enough to trigger the autolock, if it incorrectly ignored key presses.
so you can make it shorter (also please add the above to a comment so that we don't forget next time .)

Reply:
How much shorter? The timeout is 1s, below that we'll get a different letter. Something like 1.2s --> c63e7ff539930558e6dcad1faf168c2db626ef47?

Reply:
no I mean, if you want the click delay to be 1.5, you can do ceil(10/1.5) repetitions instead of 11 to get the total run time closer to the 10 (at which point an autolock would trigger if waking by keypress was incorrect)

Reply:
Ahh I see ... e233b74648687724a422b8fb5b4c9c14d209a13e?